cmake/system.cmake

```diff
@@ -102,6 +105,12 @@ if (CMAKE_C_COMPILER STREQUAL loongcc)
   set(GETARCH_FLAGS	"${GETARCH_FLAGS} -static")
 endif ()
 
+if (POWER)
+  set (NO_WARMUP 1)
+  execute_process(COMMAND bash -c "as -v < /dev/null 2>&1 | grep GNU 2>&1 >/dev/null" RESULT_VARIABLE HAVE_GAS)
```

Comment:
Shouldn't this be using `CMAKE_ASM_COMPILER` or some other cmake variable instead of `as`?

Reply:
Hi @isuruf, I modified the points you mentioned. In addition, I added an argument to force clang to generate assembly considering GNU as support. I see that the CI generated an error, but I don't think this is related to the modifications of my last commit since they are inside a scope that only enters when the variable Power is set.